packages/web-app/src/modules/profile/ProfileStore.ts

```diff
@@ -76,4 +76,27 @@ export class ProfileStore {
       // this.store.routing.replace('/')
     }
   })
+
+  @action.bound
+  updateMinecraftUsername = flow(function* (this: ProfileStore, minecraftUsername: string) {
+    if (this.currentProfile === undefined) return
+
+    this.isUpdating = true
+
+    try {
+      let patch = yield this.axios.patch('/api/v1/profile', { minecraftUsername: minecraftUsername })
```

Comment:
See the note about the API change on the Notion card. Easy fix:

```suggestion
      let patch = yield this.axios.patch('/api/v1/profile', { extensions: { minecraftUsername }})
```